sfge/src/test/java/com/salesforce/graph/vertex/LiteralExpressionVertexTest.java

```diff
@@ -0,0 +1,89 @@
+package com.salesforce.graph.vertex;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.salesforce.TestUtil;
+import com.salesforce.apex.jorje.ASTConstants.NodeType;
+import com.salesforce.graph.Schema;
+import org.apache.tinkerpop.gremlin.process.traversal.dsl.graph.GraphTraversalSource;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+public class LiteralExpressionVertexTest {
+    private GraphTraversalSource g;
+
+    // spotless:off
+    private final String isLiterallyXSourceTemplate =
+            "public with sharing class Beep {\n"
+                    + "    public void doNothing() {\n"
+                    + "        %s\n"
+                    + "    }\n"
+                    + "}";
+    // spotless:on
+
+    @BeforeEach
+    public void setup() {
+        g = TestUtil.getGraph();
+        System.out.println("pre");
+    }
+
+    @Nested
```

Comment:
Why do these need to be nested?

Reply:
Since `True` and `False` are both inner classes, it seemed logical for their tests to also be inner classes. Also it keeps the file a little cleaner, IMO.